<launch>
<arg name="nao_ip"            default="$(optenv NAO_IP)" />
<arg name="nao_port"          default="$(optenv NAO_PORT 9559)" />
<arg name="roscore_ip"        default="127.0.0.1" />
<arg name="network_interface" default="wlan0" />
<arg name="namespace"         default="naoqi_driver" />
<node name="robot_toolkit" pkg="robot_toolkit" type="robot_toolkit_node" output="screen" args="--publish_odom true --qi-url=tcp://$(arg nao_ip):$(arg nao_port) --roscore_ip=$(arg roscore_ip) --network_interface=$(arg network_interface)">
	<remap from="odom" to="odom_wheels"/>
</node>
</launch>
